webhook/server/server.go

```diff
@@ -54,6 +57,17 @@ var WebhookServerAppLabels = func() map[string]string {
 	}
 }
 
+var WebhookServerAppAnnotations = func() map[string]string {
+	//Add restart timestamp which will update the webhook server
+	//deployment to force restart. This is done so that the
+	//serviceaccount userid is updated to use the latest and the
```

Comment:
I think for SA uid is `Unique identifier` but not user id.
```suggestion
	//serviceaccount uid is updated to use the latest and the
```

Reply:
fixed!